src/stko/_internal/calculators/torsion_calculators.py

```diff
@@ -150,7 +154,11 @@ def get_results(
 
 
 class MatchedTorsionCalculator(ConstructedMoleculeTorsionCalculator):
-    """Matches rdkit generated torsions with building block torsions."""
+    """Matches rdkit generated torsions with building block torsions.
+
+    This code was developed by @joshkamm.
```

Comment:
i dont think docstrings are the appropriate place to give credit

my first choice would be authoriship on the appropriate commit or a separate contributors file. otherwise we'd have to list everyone in every docstring everywhere

Reply:
Agreed, I had an idea for giving more credit, but this is too much